src/org/labkey/test/tests/SampleTypeLinkToStudyTest.java

```diff
@@ -446,6 +448,46 @@ public void testFolderImportExport()
         checker().verifyEquals("No data should have been imported", 0, table.getDataRowCount());
     }
 
+    /*
+        Test coverage for : Issue 42937: Assay results grid loading performance can degrade with a large number of "copied to study" columns
+     */
+    @Test
+    public void testLinkedColumnDisabledCase()
```

Comment:
this test case name is a little confusing to me, what is it that is "disabled"? the other PR uses the name `testLinkedColumnNotDisplayedCase`, that one makes sense to me.

Reply:
ok will change, i was confused for name while writing the test.